File: pset6/psets/sentimental-mario-less/mario.py
from cs50 import get_int
# get the height from the user and prompt the user for wrong input
height = get_int("Height: ")
while (height <= 0 or height >= 9):
    height = get_int("Height: ")

# looing including the height starting from 1 to remove space
for i in range(1, height+1):
    print(" "*(height - i) + "#"*i)

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // getting the input from the user
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    // iterating for each row
    for (int row = 0; row < height; row++)
    {
        // creating the first hashes and spaces in front of them
        for (int space = 0; space < height; space++)
        {
            if (row + space < height - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        // printing two spaces between brick of hashes
        printf("  ");
        // printing the second brick of hashes
        for (int column = 0; column < height; column++)
        {
            if (column <= row)
            {
                printf("#");
            }
        }
        
        // taking the command line to the next line
        printf("\n");
    }
}